### PR TITLE
Add note to ECMA-402 diffs about host environment tzdata updates

### PR DESCRIPTION
**Pull request by ptomato**
Closes: #629

### CODE REVIEW

**Comment by sffc** [comment]
I don't get it.  Isn't ToLocalTime in ECMA-262?  Why are you adding it to intl.html?

**Comment by ptomato** [comment]
It is in ECMA-402: https://tc39.es/ecma402/#sec-tolocaltime

File: spec/intl.html [hunk 1]
```diff
@@ -460,6 +460,93 @@ <h1>GetDateTimeFormatPattern ( _opt_, _formats_, _hc_ )</h1>
...
+          1. Let _timeZoneOffset_ be the value calculated according to <emu-xref href="#sec-local-time-zone-adjustment">LocalTZA(_t_, *true*)</emu-xref> where the local time zone is replaced with timezone _timeZone_.
+          <ins>Given the same values of _t_ and _timeZone_, the result must be the same for the lifetime of the surrounding agent</ins>.
+          1. Let _tz_ be the time value _t_ + _timeZoneOffset_.
+          1. Return a record with fields calculated from _tz_ according to <emu-xref href="#table-temporal-datetimeformat-tolocaltime-record"></emu-xref>.
+        1. Else,
+          1. Return a record with the fields of Column 1 of <emu-xref href="#table-temporal-datetimeformat-tolocaltime-record"></emu-xref> calculated from _t_ for the given _calendar_ and _timeZone_. The calculations should use best available information about the specified _calendar_ and _timeZone_, including current and historical information about time zone offsets from UTC and daylight saving time rules.
+          <ins>Given the same values of _t_, _calendar_, and _timeZone_, the result must be the same for the lifetime of the surrounding agent</ins>.
```
Comment (justingrant) [inline spec/intl.html:480]: Why is the `<ins>` about caching not present in the Else branch?

Comment (ptomato) [reply spec/intl.html:480]: It's there in line 475 (after the call to LocalTZA)

**Review by sffc — APPROVED** [review]

File: spec/intl.html (hunk 1 above)
Comment (justingrant) [reply spec/intl.html:480]: Right, but doesn't that `<ins>` refer to the `If` branch, not the `Else` branch?

Comment (justingrant) [reply spec/intl.html:480]: Oh, never mind- it's on line 280. No idea why I didn't see that earlier.

**Review by justingrant — APPROVED** [review]

**Comment by ptomato** [comment]
Thanks for the reviews!

**Pull request merged**